.github/labels.yml

```diff
@@ -0,0 +1,49 @@
+# Main categories
+- name: 'breaking'
+  color: 'b60205'
+  description: 'Breaking API change or removal of functionality'
+- name: 'bug'
+  color: 'd73a4a'
+  description: "Something isn't working"
+- name: 'documentation'
+  color: 'a2eeef'
+  description: 'Improvements or additions to documentation'
+- name: 'feature'
+  color: '0075ca'
+  description: 'Added or improved external functionality'
+
+# Guidance
+- name: 'good first issue'
+  color: '90ee90'
+  description: 'Good for newcomers'
+- name: 'help wanted'
+  color: '008672'
+  description: 'Please help if you can'
+- name: 'question'
+  color: 'fbca04'
+  description: 'Further information is needed'
+
+# Refactoring
+- name: 'dependencies'
+  color: 'edc287'
+  description: 'Change to dependencies'
+- name: 'refactoring'
+  color: 'edc287'
+  description: 'Improvement to code without change in functionality'
+- name: 'utility'
+  color: '1d76db'
+  description: 'Changed internal functionality'
+- name: 'tests'
+  color: 'edc287'
+  description: 'Tests'
+- name: 'tooling'
+  color: 'edc287'
+  description: 'Development tools'
+
+# Triage
+- name: 'duplicate'
+  color: 'cfd3d7'
+  description: 'Duplicates another issue or pull request'
```

Comment:
_curious:_ When would we use these? (Is "triage" actually about reasons why we're discarding an issue?)

Reply:
We'd use this as a flag for when we're closing an issue 👍

Reply:
Aaaaah I see thank you 👍